src/main/java/com/minecraftabnormals/abnormals_core/core/registry/ACLootConditions.java

```diff
@@ -0,0 +1,21 @@
+package com.minecraftabnormals.abnormals_core.core.registry;
+
+import com.minecraftabnormals.abnormals_core.core.AbnormalsCore;
+import com.minecraftabnormals.abnormals_core.core.api.conditions.loot.RaidCheckCondition;
+import com.minecraftabnormals.abnormals_core.core.api.conditions.loot.RandomDifficultyChanceCondition;
+import net.minecraft.loot.LootConditionType;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.registry.Registry;
+
+/**
+ * @author abigailfails
+ * */
+public class ACLootConditions {
```

Comment:
There could be more documentation about this class's purpose.

Reply:
Is what i just committed the right direction?

Reply:
Changed it again based on your suggestion.